tests/tests_sign_workflow/test_sign_artifacts.py

```diff
@@ -55,29 +57,37 @@ def test_sign_with_build_manifest(self):
             'maven/org/opensearch/opensearch-performance-analyzer/1.1.0.0/opensearch-performance-analyzer-1.1.0.0.jar',
             'maven/org/opensearch/opensearch-performance-analyzer/1.1.0.0/opensearch-performance-analyzer-1.1.0.0-sources.jar'
         ]
-        signer.sign_artifacts.assert_called_with(expected, BUILD_MANIFEST.parent, sigtype)
+        signer.sign_artifacts.assert_called_with(expected, manifest.parent, sigtype)
 
     def test_sign_existing_artifacts_file(self):
         path = Path(r"/dummy/path/file.tar.gz")
         sigtype = '.sig'
         signer = MagicMock()
-        signer_with_manifest = SignArtifactsExistingArtifactFile(target=path,
-                                                                 component='maven',
-                                                                 artifact_type='dummy',
-                                                                 signature_type=sigtype,
-                                                                 signer=signer)
+        signer_with_manifest = SignArtifactsExistingArtifactFile(
+            target=path,
+            component='maven',
+            artifact_type='dummy',
+            signature_type=sigtype,
+            signer=signer
+        )
         signer_with_manifest.sign()
         signer.sign_artifact.assert_called_with("file.tar.gz", path.parent, sigtype)
 
-    def test_sign_existing_artifacts_folder(self):
-        path = Path(os.path.join(os.path.dirname(__file__), "data/artifacts"))
+    @patch('os.walk')
+    def test_sign_existing_artifacts_folder(self, mock_os_walk):
+        mock_os_walk.return_value = [
+            ('dummy', (), ['tar_dummy_artifact_1.0.0.tar.gz', 'zip_dummy_artifact_1.1.0.zip'])
```

Comment:
The documentation define the second return of `os.walk` is a list `[]`, while you mock to a taple `()`, will this be a problem after all? Just curious.

https://docs.python.org/2/library/os.html#os.walk